it-and-security/lib/windows/scripts/migrate-windows-ca-2023.ps1

```
@@ -0,0 +1,170 @@
+
+$migratedCount = 0
+$missingCount  = 0
+foreach ($f in $bootFiles) {
+    $name = Split-Path $f -Leaf
+    if (-not (Test-Path $f)) {
+        Write-State $name "MISSING"
+        $missingCount++
+        continue
+    }
+    $issuer = $null
+    try { $issuer = (Get-AuthenticodeSignature $f).SignerCertificate.Issuer } catch {}
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)